sourcecode/hub/app/Http/Requests/ContentFilter.php

```diff
@@ -348,8 +347,8 @@ private function attachModel(array $hits, bool $forUser, bool $showDrafts): Coll
                 previewUrl: route('content.preview', [$model, $version]),
                 useUrl: $canUse ? route('content.use', [$model, $version]) : null,
                 editUrl: $canEdit ? route('content.edit', [$model, $version]) : null,
-                shareUrl: $canView ? route('content.share', [$model, SessionScope::TOKEN_PARAM => null]) : null,
-                copyUrl: $canCopy ? route('content.copy', [$model, $version]) : null,
+                shareUrl: $canView ? route('content.share', [$model]) : null,
```

Comment:
Why the removal of `SessionScope::TOKEN_PARAM => null`?

Reply:
Thought it wasn't required since on the other two uses of the route one has `SessionScope::TOKEN_PARAM => null` the other doesn't

Reply:
I think it's actually the usage without it that's mistaken. But in that case, there's a handler intercepting clicks on the link anyway.

Reply:
I've restored the parameter